### PR TITLE
[VSFeat] Monitoring View Implementation

### PR DESCRIPTION
**Pull request by ccastrotrejo**
### Main Code Change
- Add Intl provider to vscode designer
- Add query client provider to vscode designer
- Implementation of Resubmit button in monitoring view
- Implementation of Refresh button
- Change implementation of getting run instance into react query
- Add loading and error scenario for vscode designer
- Implementation of try catch to run service

### Loading Scenario
Screenshot 2023-03-03 at 7 34 25 PM

### Error Scenario
Screenshot 2023-03-03 at 7 32 18 PM

### Refresh on local 

https://user-images.githubusercontent.com/102700317/222870796-e73a0dfe-f65e-4a37-9674-ea17bb8cdc22.mov

### Refresh on remote

https://user-images.githubusercontent.com/102700317/222870790-c3cafab2-b3d1-4553-903e-8468e96b55b4.mov

### CODE REVIEW

File: apps/vs-code-designer-react/src/app/app.tsx [hunk 1]
```diff
@@ -30,9 +34,21 @@ export const App = () => {
     isMonitoringView,
     runId,
   } = vscodeState;
-  const [standardApp, setStandardApp] = useState(panelMetaData?.standardApp);
+  const [standardApp, setStandardApp] = useState<StandardApp | undefined>(panelMetaData?.standardApp);
   const [runInstance, setRunInstance] = useState<LogicAppsV2.RunInstanceDefinition | null>(null);
   const [theme, setTheme] = useState<Theme>(getTheme(document.body));
+  const intl = useIntl();
+
+  const intlText = {
```
Comment (ccastrotrejo) [inline apps/vs-code-designer-react/src/app/app.tsx:42]: Hi @ecfan these are the only text I added, both of them have screenshots in the PR description

**Review by Eric-B-Wu — APPROVED** [review]

File: apps/vs-code-designer-react/src/app/app.tsx (hunk 1 above)
Comment (ecfan) [reply apps/vs-code-designer-react/src/app/app.tsx:42]: @ccastrotrejo: I noticed the following text, which shouldn't use the word "please" [except in specific cases](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/please):

image

Comment (hartra344) [reply apps/vs-code-designer-react/src/app/app.tsx:42]: @ecfan That isn't our text, that is the text that Apple includes on the terminal to let you know it's ZSH, separate from us entirely

Comment (ecfan) [reply apps/vs-code-designer-react/src/app/app.tsx:42]: 👍🏻😢

**Review by ecfan — APPROVED** [review]

**Pull request merged**